docs/documentation/modules/circuit.py

```diff
@@ -0,0 +1,248 @@
+from __future__ import annotations
```

Comment:
@dominikandreasseitz are all the files in this folder needed?

Reply:
@dominikandreasseitz  ok thanks 
@madagra  yes  , I think the system is setup to look inside docs folder  to generate documentation  , so the documentation will end up showing  showing like this   on the site https://drive.google.com/file/d/11N4TCD4wIvp_E56DcD3PpIznuUl_d6qV/view?usp=sharing

Reply:
@madagra is right. we dont need these files here,  thats what the mkdocs.yml is for. just try to replicate what we did there. sorry for not spotting this earlier! do you think you can fix it today @prilcool ?

Reply:
@dominikandreasseitz   ok  I can remove those files  from the docs folder ,  but from what i understand  the  mkdocs.yml   by default  is configured for searching only the docs folder   so   currently my entry in the mcdocs.yml is  which works 

  - Documentation:
    - Modules:
        - Circuit : documentation/modules/circuit.py
        - Matrices : documentation/modules/matrices.py
        - Parametric : documentation/modules/parametric.py
        - Primitive : documentation/modules/primitive.py


But when  I   have tried  to update the entry to the below  after remove  the files from the docs folder   and specifying for it to look into  pyqtorch/modules  It doesn't work   as it search's for the files in the docs folder .

  - Documentation:
    - Modules:
        - Circuit : pyqtorch/modules/circuit.py
        - Matrices : pyqtorch/matrices.py
        - Parametric : pyqtorch/modules/parametric.py
        - Primitive : pyqtorch/modules/primitive.py

Reply:
in `docs`, add a circuit.md file with the following content
`## Cirucit

### ::: pyqtorch.modules.circuit`

it should automatically see the docstrings.

Reply:
and in the mkdocs.yml just reference `circuit.md`

Reply:
@dominikandreasseitz  ah thanks , learnt something new ,  
Should be  good now    
I made the update commits 
[89f0c2b](https://github.com/pasqal-io/PyQ/pull/72/commits/89f0c2bd267fd20345f1ccb9c231238b76eb2213)

Reply:
lint breaks, can u fix and resubmit?

Reply:
@dominikandreasseitz   ok here   [d859c71](https://github.com/pasqal-io/PyQ/pull/72/commits/d859c718938b29749b407c60eb707914080fe9c6)